### ISSUE TITLE
Exclude HTTPS URLs When Checking

### ISSUE THREAD

**Issue by praneesha** [peter-evans/link-checker#38]
@peter-evans - I have fixed all the broken links and now I get only the error below in my report.

<img width="662" alt="Screenshot 2021-01-12 at 17 41 33" src="https://user-images.githubusercontent.com/11707273/104313169-a5cbfb80-54fd-11eb-9fa1-3ef6501b6527.png">

How can I get rid of this to get an empty report? My config is as follows:

```
on: [pull_request]
name: Link Checker Example Command
jobs:
  linkChecker:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v2
      - name: Build the site
        uses: shafreenAnfar/jekyll-build-action@v5
      - name: Link Checker
        id: lc
        uses: peter-evans/link-checker@v1
        with:
          args: -r learn swan-lake 1.1 1.0 -x 0.990 https?:// -d ./_site 
      - name: Archive production artifacts
        uses: actions/upload-artifact@v2
        with:
          name: Link Checker Report
          path:  ./link-checker/out.md
      - name: Fail if there were link errors
        run: exit ${{ steps.lc.outputs.exit_code }}
```

**Comment by praneesha** [comment]
@peter-evans - Any answer on the above, please?

**Comment by peter-evans** [comment]
@praneesha It looks like there is something wrong with your `args`. The error message is complaining about the `https?://` part.

By the way, [Liche](https://github.com/raviqqe/liche) was recently deprecated and as a result I've also decided to deprecate this action in favour of [lychee-action](https://github.com/lycheeverse/lychee-action), which is a fork of this project based on [lychee](https://github.com/lycheeverse/lychee). Please consider using that action instead.

**Comment by praneesha** [comment]
@peter-evans - Sorry, The Lychee tool doesn't work for me (as reported in https://github.com/lycheeverse/lychee/issues/106) and I am almost done with configuring your tool.

Thereby, is there a way to skip HTTPS URLs without giving it in the args as I have done above?
